run.bat

```diff
@@ -1,3 +1,7 @@
 echo off
 cmake --build .
-start \c++\game\game\bin\Debug\game.exe
\ No newline at end of file
+start \c++\game\game\bin\Debug\game.exe
+del \c++\game\game\*.vcxproj
+del \c++\game\game\*.filters
+del \c++\game\game\*.cmake
+del \c++\game\game\CMakeCache.txt
```

Comment:
Maybe it would be better to build project in a different directory and just 

```batchfile
rmdir /s directory_name
``` 
 at the end